crates/holochain/src/core/ribosome/host_fn/create.rs

```diff
@@ -19,7 +19,7 @@ pub fn create<'a>(
     // build the entry hash
     let async_entry = AsRef::<Entry>::as_ref(&input).to_owned();
     let entry_hash =
-        holochain_types::entry::EntryHashed::from_content_sync(async_entry).into_hash();
+        holochain_zome_types::entry::EntryHashed::from_content_sync(async_entry).into_hash();
```

Comment:
This should be:
```rust
let entry_hash = EntryHash::with_data(AsRef::<Entry>::as_ref(&input)));
```
Because the `to_owned` is cloning the entry for no reason.